lib/services/local/users.go

```diff
@@ -1441,17 +1441,7 @@ func (s *IdentityService) UpsertKeyAttestationData(ctx context.Context, attestat
 		return trace.Wrap(err)
 	}
 
-	pub, err := x509.ParsePKIXPublicKey(attestationData.PublicKeyDER)
```

Comment:
We probably want to keep this to make sure the key is valid before writing to storage. Let's make sure tests cover invalid keys as well.